src/shared/components/home/instances.tsx

```diff
@@ -133,7 +134,7 @@ export class Instances extends Component<any, InstancesState> {
     );
   }
 
-  itemList(items: Instance[]) {
+  itemList(items: Instance[], link?: boolean | undefined) {
```

Comment:
Since you're checking if `link` is true or undefined, it is cleaner to use a default param here:
```suggestion
  itemList(items: Instance[], link = true) {
```